terraform/main.tf

```diff
@@ -84,7 +84,10 @@ resource "null_resource" "apply_deployment" {
 resource "null_resource" "wait_conditions" {
   provisioner "local-exec" {
     interpreter = ["bash", "-exc"]
-    command     = "kubectl wait --for=condition=ready pods --all -n ${var.namespace} --timeout=-1s"
+    command     = <<-EOT
+    kubectl wait --for=condition=AVAILABLE apiservice/v1beta1.metrics.k8s.io --timeout=1200s
+    kubectl wait --for=condition=ready pods --all -n ${var.namespace} --timeout=1200s
```

Comment:
Is 20 minutes maybe a bit too long, or is it expected that it may take as long?

Reply:
Good thought.
From [my local testing](https://screenshot.googleplex.com/6AkrRfGHRnUYDUY.png):
* waiting on `apiservice/v1beta1.metrics.k8s.io` took about 90s
*waiting on all Pods took about 140s
So I've doubled each value above and updated the `--timeout`s:
```
kubectl wait --for=condition=AVAILABLE apiservice/v1beta1.metrics.k8s.io --timeout=180s
kubectl wait --for=condition=ready pods --all -n ${var.namespace} --timeout=280s
```